docs/modules/ROOT/partials/ref-registry-pod-spec.adoc

```diff
@@ -0,0 +1,88 @@
+[#pod-spec]
+// Do not forget to update link text in related xref(s). Antora does not support automatic name if the link has a fragment.
+= Configuring {registry} deployment using `PodTemplate`.
+
+WARNING: This is a preview feature, which may evolve in the future releases.
```

Comment:
```suggestion
WARNING: This is a Technology Preview feature, which may evolve in the future releases.
```

Reply:
Suggest adding the following instead for downstream docs:  

ifdef::service-registry[]
IMPORTANT: This is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production.
            
These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview.
endif::[]